platforms/documentation/docs/build.gradle

```diff
@@ -38,7 +38,12 @@ configurations {
     docsTestRuntimeClasspath.extendsFrom(integTestDistributionRuntimeOnly)
 }
 
+configurations.configureEach {
+    exclude([group: "org.slf4j", module: "slf4j-simple"])
```

Comment:
Can we add a comment on why we're excluding this? Where it comes from and maybe why we're not using component metadata rules?